tests/format/scss/scss/scss.scss

```diff
@@ -1090,6 +1090,18 @@ div {
   margin: - pow(2, 2) * 100px;
 }
 
+div {
+  margin: -double(-double(1em));
+  margin: -double(- double(1em));
+  margin: - double(-double(1em));
+}
+
+@function asdf() {
+  @return (
+    width: -double(1em);
+  )
+}
```

Comment:
```suggestion
@function asdf() {
  @return width: -double(1em);
}
```

Reply:
Changed it https://github.com/prettier/prettier/pull/15370/commits/3f9921e94db056236ff5a32c583ce37023ae698d